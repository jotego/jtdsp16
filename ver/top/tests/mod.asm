# Test modular increment
r0 = 0x1000
rb = 0x1000
re = 0x1003
*r0++=r0
*r0++=r0
*r0++=r0
*r0++=r0
*r0++=r0
*r0++=r0
*r0++=r0
*r0++=r0
*r0++=r0
*r0++=r0
end:
goto end
