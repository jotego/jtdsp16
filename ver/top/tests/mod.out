r0=0x1002
r1=0x0000
r2=0x0000
r3=0x0000
rb=0x1000
re=0x1003
j =0x0000
k =0x0000
pr=0x0000
pt=0x0000
